dotCMS/src/main/java/com/liferay/util/FileUtil.java

```diff
@@ -199,12 +200,15 @@ public static void copyFile(File source, File destination, boolean hardLinks) th
                     Logger.warn(FileUtil.class, "Can't create hardLink. source: " + source.getAbsolutePath()
                             + ", destination: " + destination.getAbsolutePath());

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ", destination: " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)